deployment/ansible/group_vars/all

```diff
@@ -1,12 +0,0 @@
----
```

Comment:
Agreed, this didn't feel better than just templating `settings.py` directly.  Not sure if separately templating each of the environment `group_vars` files is ideal either, but ok, that's an improvement.